darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1300,6 +1300,36 @@ def save_model(self, path: str) -> None:
         # TODO: the parameters are saved twice currently, once with complete
         # object, and once with PTL checkpointing.
 
+        if path is None:
+            # default path
+            path = f"{type(self).__name__}_{datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}.pt"
+
+        # We save the whole object to keep track of everything
+        with open(path, "wb") as f_out:
+            torch.save(self, f_out)
+
+        # In addition, we need to use PTL save_checkpoint() to properly save the trainer and model
+        if self.trainer is not None:
+            path_ptl_ckpt = path + ".ckpt"
+            self.trainer.save_checkpoint(path_ptl_ckpt)
+
+    def save_model(self, path: str) -> None:
+        """
+        `TorchForecastingModel.save_model()` is deprecated and will be removed in a future darts version.
```

Comment:
You can make this look nicer using a numpydoc deprecationwarning, something like this:
```
.. deprecated:: 0.21.0
          `TorchForecastingModel.save_model()` is deprecated and will be removed in a future darts version.
          Please use `TorchForecastingModel.save()` instead.
```